gevent/_sslgte279.py

```diff
@@ -575,6 +601,18 @@ def version(self):
             return None
         return self._sslobj.version()
 
+if PYPY or not hasattr(SSLSocket, 'timeout'):
+    # PyPy (and certain versions of CPython) doesn't have a direct
```

Comment:
With that line disabled, CPython 2.7.9 fails a test like this:

```python
| //python -u test___example_servers.py
  + //python -u webproxy.py
  Serving on :8088...
  127.0.0.1 - - [2015-04-14 11:52:06] "GET / HTTP/1.1" 200 475 0.000353
  http://www.google.com: 200 OK
  127.0.0.1 - - [2015-04-14 11:52:07] "GET /http://www.google.com HTTP/1.1" 200 20096 0.202283
  .+ //bin/python -u wsgiserver.py
  Serving on 8088...
  127.0.0.1 - - [2015-04-14 11:52:10] "GET / HTTP/1.1" 200 119 0.000467
  127.0.0.1 - - [2015-04-14 11:52:10] "GET /xxx HTTP/1.1" 404 126 0.000128
  .+ //bin/python -u wsgiserver_ssl.py
  Serving on https://127.0.0.1:8443
  Traceback (most recent call last):
    File "//gevent/gevent/pywsgi.py", line 519, in handle_one_response
      self.run_application()
    File "//gevent/gevent/pywsgi.py", line 506, in run_application
      self.process_result()
    File "//gevent/gevent/pywsgi.py", line 497, in process_result
      self.write(data)
    File "//gevent/gevent/pywsgi.py", line 414, in write
      self._write_with_headers(data)
    File "//gevent/gevent/pywsgi.py", line 432, in _write_with_headers
      self._sendall(towrite)
    File "//gevent/gevent/pywsgi.py", line 389, in _sendall
      self.socket.sendall(data)
    File "//gevent/gevent/_sslgte279.py", line 397, in sendall
      v = self.send(data[count:])
    File "//gevent/gevent/_sslgte279.py", line 351, in send
      timeout = self.timeout
  AttributeError: 'SSLSocket' object has no attribute 'timeout'
```

The comment is attempting to explain that sometimes SSLSocket is used to wrap a raw/native socket (_socket.socket), sometimes it wraps a `gevent._socket2` socket. The former does not have a `timeout` property.